ACORN/current_generator/acorn_utils.py

```diff
@@ -15,7 +15,7 @@ def __getattr__(self, name):
             return name
         raise AttributeError
 
-ACORNError = Enum(["NOT_ENOUGH_FILES", "ERROR", "SUCCESS"])
+ACORNError = Enum(["NOT_ENOUGH_FILES", "NO_CURRENT_DATA", "ERROR", "SUCCESS"])
```

Comment:
According to the [coding conventions](https://github.com/aodn/data-services/issues/283) this module constant should be named `ACORN_ERROR`.
Also, considering "SUCCESS" is one of the options, maybe `ACORN_STATUS` would be a better name.